packages/fxa-settings/src/models/Account.ts

```diff
@@ -133,6 +133,62 @@ export interface ProfileInfo {
   emails: Email[];
 }
 
+// Account attributes that can be persisted
+const PERSISTENT = {
+  accountResetToken: undefined,
+  alertText: undefined,
+  displayName: undefined,
+  email: undefined,
+  grantedPermissions: undefined,
+  hadProfileImageSetBefore: undefined,
+  lastLogin: undefined,
+  // This property is set when a user has changed their primary email address and
+  // attempts to login. Similar to logging in with a different email capitalization
+  // the auth-server will return the proper email to reattempt the login. When reattempting
+  // to login, this needs to be passed back to the auth-server.
+  originalLoginEmail: undefined,
+  // password field intentionally omitted to avoid unintentional leaks
+  permissions: undefined,
+  profileImageId: undefined,
+  profileImageUrl: undefined,
+  profileImageUrlDefault: undefined,
+  providerUid: undefined,
+  recoveryKeyId: undefined,
+  sessionToken: undefined,
+  // Hint for future code spelunkers. sessionTokenContext is a misnomer,
+  // what the field is really used for is to indicate whether the
+  // sessionToken is shared with Sync. It will be set to `fx_desktop_v1` if
+  // the sessionToken is shared. Users cannot sign out of Sync shared
+  // sessions from within the content server, instead they must go into the
+  // Sync panel and disconnect there. The reason this field has not been
```

Comment:
I don't believe this is true anymore. A year ago or so? Ana (Sync product manager at the time) made the call that we should allow users to sign out from Settings which would also disconnect Sync. Related, #18029, where I changed only allowing this with the Sync flow to any flow.